mongoose.c

```diff
@@ -1363,12 +1363,27 @@ struct mg_connection *mg_http_listen(struct mg_mgr *mgr, const char *url,
 #endif
 
 
+#ifdef MG_SECURE_MEMORY
+static void *memset_secure(void *v, int c, size_t n)
```

Comment:
How is that different from a standard `memset()` ?

Reply:
See [PVS warning](https://www.viva64.com/en/w/v597/).
But why not reuse memset_s() and co if available?

Reply:
it would definitely be preferable to use memset_s where available, but I'm not sure how widespread compiler support for C11 is and don't know how much complexity it would add in terms of checking the C standard etc., hence going for the simplest possible approach